api/src/data/models/action-attachment-model.ts

```diff
@@ -0,0 +1,10 @@
+export interface ActionAttachment {
```

Comment:
Whenever possible match the file name and the default export of the file.

Normally models are just named `/models/action-attachment.ts` and everything else is named with a type. e.g. "/services/action-attachments/create-service". This makes the most common reference, the model, the easiest to find via editor search.